### ISSUE TITLE
Issue: OpenTelemetry quick start mentions OTLP/OTel exporters which I believe are deprecated

### ISSUE THREAD

**Issue by adiosspandit** [newrelic/docs-website#5764]
## How can we make our docs better?

* New Relic exporters are documented as being deprecated (https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters/)

* One of the OpenTelemetry steps still mentions using an exporter (https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start/#export)
Just wondering if this should be removed and we should just mention that OTel Collector Agent can directly send data to New Relic OTel endpoints.

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start/
* **OS**: Mac OS 10.15
* **Browser**: Firefox 94.0
* **Device**: Unknown

**Comment by rhetoric101** [comment]
@adiosspandit, we recently updated the legacy exporters document (https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters), showing that all the New Relic exporters are archived, except the collector. That exporter is due to be archived by May 1, 2022. This was updated by an SME who is close to the project. Our plan is to remove this page once the final legacy exporters is removed. Does that raise concerns for you?
